testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveSparkCompatibility.java

```diff
@@ -645,4 +647,207 @@ public void testReadSparkBucketedTable()
 
         onSpark().executeQuery("DROP TABLE " + sparkTableName);
     }
+
+    @Test(groups = {HIVE_SPARK, PROFILE_SPECIFIC_TESTS})
+    public void testReadSparkStatisticsUnpartitionedTable()
+    {
+        String tableName1 = "test_trino_reading_spark_statistics_table_" + randomNameSuffix();
+        String tableName2 = "test_trino_reading_spark_statistics_table_" + randomNameSuffix();
+        try {
+            onSpark().executeQuery(format("CREATE TABLE %s(" +
+                    "c_tinyint            BYTE, " +
+                    "c_smallint           SMALLINT, " +
+                    "c_int                INT, " +
+                    "c_bigint             BIGINT, " +
+                    "c_float              REAL, " +
+                    "c_double             DOUBLE, " +
+                    "c_decimal            DECIMAL(10,0), " +
+                    "c_decimal_w_params   DECIMAL(10,5), " +
+                    "c_timestamp          TIMESTAMP, " +
+                    "c_date               DATE, " +
+                    "c_string             STRING, " +
+                    "c_varchar            VARCHAR(10), " +
+                    "c_char               CHAR(10), " +
+                    "c_boolean            BOOLEAN, " +
+                    "c_binary             BINARY" +
+                    ")", tableName1));
+
+            onSpark().executeQuery(format("INSERT INTO %s VALUES " +
+                    "(120, 32760, 2147483640, 9223372036854775800, 123.340, 234.560, CAST(343.0 AS DECIMAL(10, 0)), CAST(345.670 AS DECIMAL(10, 5)), TIMESTAMP '2015-05-10 12:15:30', DATE '2015-05-08', 'p1 varchar', CAST('varchar10' AS VARCHAR(10)), CAST('p1 char10' AS CHAR(10)), false, CAST('p1 binary' as BINARY))," +
```

Comment:
It would be beneficial to add in this test another row with non-null values.

As can be seen from here `io.trino.plugin.hive.metastore.thrift.ThriftSparkMetastoreUtil#fromMetastoreDistinctValuesCount(long, long, long)` the ndv count is set to `1` so a future maintainer of the code may be a bit puzzled to see that the NDV doesn't correspond to the number of actual distinct values.